### PR TITLE
Update layout for register account

### PR DESCRIPTION
**Pull request by MikeHoang**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -47,6 +47,9 @@ class User < ActiveRecord::Base
   #validates :profile, presence: true
   enum user_type: { admin: 'Admin', partner: 'Partner', student: 'Student', event_editor: 'EventEditor', blog_edit: 'BlogEditor', student_hero: 'StudentHero', pending: 'Pending' }
   enum sex: { male: 0, female: 1}
+  enum payment_type: { bank: 0, cash: 1}
+  enum room_type: { share: 0, single: 1, master: 1}
```
Comment (nguyenductoan) [inline app/models/user.rb]: Are single and master the same type of room?

File: app/controllers/admin/students_controller.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def show
 
   def create
     @student = Student.new(student_params)
-    @student.password = Faker::Internet.password(User::PASSLENGTH[:min], User::PASSLENGTH[:max])
+    @student.password = SecureRandom.hex(8)
```
Comment (lhhieu89) [inline app/controllers/admin/students_controller.rb]: Please use `Devise.friendly_token[0, 8]` instead

File: app/controllers/partner/promotions_controller.rb [hunk 3]
```diff
@@ -30,17 +30,34 @@ def edit
   end
 
   def create
-    @promotion = Promotion.new(promotion_params)
-    @promotion.save
-    redirect_to new_promotion_path
+    @partner = Partner.new(partner_params)
+    @partner.user_type = "Pending"
+    @partner.approve_type = "Partner"
+    @partner.password = SecureRandom.hex(8)
```
Comment (lhhieu89) [inline app/controllers/partner/promotions_controller.rb]: Please use `Devise.friendly_token[0,8]`

**Comment by lhhieu89** [comment]
Please remove `public/uploads` and fix the confict

**Pull request merged**